style.css

```diff
@@ -32,106 +29,122 @@ hr {
 .wrapper{
 	height: 80vh;
 	display: grid;
-	grid-template-rows: 200px 15px 250px 250px 250px 5%;
-	grid-template-columns: 1fr 2fr 2fr 2fr 2fr 1fr; /*Left, Boxes Left x2, Boxes Right x2, Right*/
+	grid-template-rows: 4vw 49vw 12vw 12vw;
+	grid-template-columns: auto;
+ }
+
+.wrapper2{
+	height: 80vh;
+	display: grid;
+	grid-template-rows: 4vw 58vw 12vw;
+	grid-template-columns: auto;
  }
 
  /*----------------BURGER NAV---------------------*/
+ 
+ .nav{
+	box-sizing: border-box;
+	background-color: #f7f7f7;
+	padding-left: 1vw;
+	padding-right: 1vw;
+	background: rgba(204, 204, 204, 0.7);
+	grid-column: 1 / span 6;
+	grid-row: 1;
+}
 
 #menu__toggle {
-	  opacity: 0;
+	opacity: 0;
 }
 
 #menu__toggle:checked + .menu__btn > span {
-	  transform: rotate(45deg);
+	transform: rotate(45deg);
 }
 
 #menu__toggle:checked + .menu__btn > span::before {
-	  top: 0;
-	  transform: rotate(0deg);
+	top: 0;
+	transform: rotate(0deg);
 }
 
 #menu__toggle:checked + .menu__btn > span::after {
-	  top: 0;
-	  transform: rotate(90deg);
+	top: 0;
+	transform: rotate(90deg);
 }
 
 #menu__toggle:checked ~ .menu__box {
-	  left: 0 !important;
+	left: 0 !important;
 }
 
 .menu__btn {
-	  position: fixed;
-	  top: 40px;
-	  left: 20px;
-	  width: 26px;
-	  height: 26px;
-	  cursor: pointer;
-	  z-index: 3;
+	position: absolute;
+	top: 2vw;
+	left: 1vw;
+	width: 1.8vw;
+	height: 2vw;
+	cursor: pointer;
+	z-index: 3;
 }
 
 .menu__btn > span,
 .menu__btn > span::before,
 .menu__btn > span::after {
-	  display: block;
-	  position: absolute;
-	  width: 100%;
-	  height: 3px;
-	  background-color: #616161;
-	  transition-duration: .30s;
+	display: block;
+	position: absolute;
+	width: 100%;
+	height: 0.25vw;
+	background-color: #616161;
+	transition-duration: .30s;
 }
 
 .menu__btn > span::before {
-	  content: '';
-	  top: -8px;
+	content: '';
+	top: -8px;
 }
 
 .menu__btn > span::after {
-	  content: '';
-	  top: 8px;
+	content: '';
+	top: 8px;
 }
 
 .menu__box {
-	  display: block;
-	  position: fixed;
-	  top: 0;
-	  left: -100%;
-	  width: 300px;
-	  height: 100%;
-	  margin: 0;
-	  padding: 80px 0;
-	  list-style: none;
-	  background-color: #ECEFF1;
-	  box-shadow: 0px 0px 0px rgba(0, 0, 0, .4);
-	  transition-duration: .30s;
+	display: block;
```

Comment:
Here I have changed the position to absolute instead of it being fixed so it will work better for the menu_box. I also changed the width, height and padding from pixels to now become view port width (vw).